interpreter.js

```diff
@@ -2807,8 +2807,24 @@ Interpreter.prototype['stepCallExpression'] = function() {
   if (!state.doneExec_) {
     state.doneExec_ = true;
     var func = state.func_;
-    if (!func || !func.isObject) {
-      this.throwException(this.TYPE_ERROR, func + ' is not a function');
+    var isNotFunction = ' is not a function';
```

Comment:
nit; I'd recommend calling this something like `nonCallableErrorMessage`. Names should represent the _why_ of their contents, not the _what_

http://www.thecodelesscode.com/case/95

Reply:
Thank you! I will use this style going forward. In this case, a refactor eliminated the need for it.